Cold-Gas-Thruster/Cold Gas Thruster PDD.tex

```diff
@@ -0,0 +1,462 @@
+
+This concludes the milestones for this project.
+
+\section{Externalities}
+  % Things not directly related to the work or outcomes, but related to the project as a whole.
+\subsection{Prerequisite Skills}
+  % Which skills do team members need to have before work can start (not including skills that will be learned ``on the job'')?
+
+Creating a CGT will require a few skills, many of which can be learned throughout the project. The design process will require knowledge of systems and how they
+work together, basic nozzle theory, computer-aided design (CAD), basic electrical engineering, basic coding abilities, and machining. The majors that would
+be most qualified for satisifing these requirements are mechanical engineering \& technology and electrical engineering \& technology. But this project is certainly
+open to all majors because most of these skills can be taught in at least a basic manner.
+
+The basic nozzle theory required here is laid out in several documents
+that are on the google drive, the best one being Nozzles.pdf. It is very consise and clear. The beginner propulsion project also lays out a lot of the basic theory in
+relation to minimum length converging diverging nozzles.
+
+ The CAD software of choice is Solidworks. It is very common for industry and has all the features that the team
+will need throughout the process. The education edition also includes the simulation premium package that can allow the team to run flow simulations if needed. The procedure for
+doing this is complicated but can be learned to a basic degree on youtube.
+
+The basic electrical engineering including setting up and arduino or raspberry pi to read out the data from the load cell and turning the solenoid on and off. It
+should not be difficult for a first year or second year EE or EET to set this up. If it proves to be difficult, the team can each out to older students or professors for assistance.
+Dr. Patru would certainly be a great resource for this kind of help.
+
+The basic coding abilites include writing a MATLAB script to analyze and plot data. This is not hard for anyone who has coded before. If it proves to be a roadblock
+one can look up the MATLAB scripts already in the google drive for reference. These scripts layout all required techniques and a bit more, such as an iterative approach.
+
+The machining will be the most difficult to learn if there are no MEs or METs that have taken the proper courses. Machining can be extremely dangerous and cannot be done
+without the proper safety training. If no one is trained on the machines one can ask a machinist to make simple parts or submit a job at the Brickmann lab, formula machine shop.
+They have professional engineers there that can make nearly anything one could need. But their timeline slips often, so leave an appropriate amount of time for them to make the part
+so it does not interfere with the projects timeline.
```

Comment:
For the preceding sections, you should list the required skills, whether members should have those skills before starting the project or whether they can be learned in-situ, and specify where outside help is needed.

Be concise. This section is a resource for reviewers to see exactly what manpower and skill sets are required for a successful project.